pycontrails/core/met.py

```diff
@@ -1218,13 +1312,18 @@ def __init__(
         wrap_longitude: bool = False,
         copy: bool = True,
         validate: bool = True,
+        name: Hashable | None = None,
         **kwargs: Any,
     ) -> None:
         # init cache
         self.cachestore = cachestore
 
         # try to create DataArray out of input data and **kwargs
         if not isinstance(data, xr.DataArray):
+            DeprecationWarning(
```

Comment:
How should we tag these for revision in the future?

Reply:
I don't know! We could say that it'll be removed in the next minor version update ... does this sound right to you? (In this particular example, I think we _could_ just remove it right away.)